src/Orleans.Runtime/Catalog/ActivationData.cs

```diff
@@ -806,6 +802,19 @@ void RecordRunning(Message message, bool isInterleavable)
                 while (true);
             }
 
+            void RecordRunning(Message message, bool isInterleavable)
```

Comment:
This local function was moved outside of the block to reduce nesting, making the diff look a little strange